tests/fused_attention_stablehlo_test.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2023 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from absl.testing import absltest
+from typing import Any, Optional
+import os
+os.environ['XLA_FLAGS'] = '--xla_gpu_enable_cudnn_fmha=true --xla_gpu_fused_attention_use_cudnn_rng=true'
+
+import numpy as np
+import jax
+import jax.numpy as jnp
+from jax.sharding import Mesh
+from jax.sharding import PartitionSpec, NamedSharding
+from jax._src import config
+from jax._src import test_util as jtu
+from jax._src.cudnn.fused_attention_stablehlo import dot_product_attention
```

Comment:
Is that how we want people to import it?
I think something like: jax.cudnn.... would be better.
We also need the new operation to show up in the documentation.

Reply:
I added a package under jax.cudnn and import from jax._src.
Regarding the docs, where would be best to put the doc?

Reply:
@yashk2810 Any idea about the doc?